compiler/pre_hook_amd-llvm.cmake

```
@@ -35,6 +35,13 @@ else()
     # For now we will switch the priorty for find_package to first search in
     # CONFIG mode.
     set(RUNTIMES_CMAKE_ARGS "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON")
+
+    # We may need to guard this for amd-staging only builds.
```

```suggestion
    # TODO: We may need to guard this for amd-staging only builds.
```
Maybe make this a TODO?